.github/workflows/tests.yml

```
@@ -0,0 +1,38 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - ${{ github.event.repository.default_branch }}
+  pull_request:
+    branches:
+      - ${{ github.event.repository.default_branch }}
+    paths:
+      - '*.js'
+      - 'package.json'
+
+jobs:
+  tests:
+    env:
+      NPM_CONFIG_PROGRESS: "false"
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 6
+      - run: npm install --no-audit && npm install karma-cli karma-sauce-launcher
```

The second `npm install` should probably include `--no-audit` as well.

```suggestion
      - run: npm install --no-audit && npm install --no-audit karma-cli karma-sauce-launcher
```